tests/App/AppZipperTest.php

```diff
@@ -19,24 +20,30 @@ class AppZipperTest extends TestCase
 
     public function setUp(): void
     {
-        $this->tempDir = Path::join((string) realpath(sys_get_temp_dir()), $this->name());
+        $this->tempDir = Path::join((string) realpath(sys_get_temp_dir()), $this->name(), 'temp');
         $this->fs = new Filesystem();
     }
 
     public function testAppZipperAddsAllFiles(): void
     {
-        $zipper = new AppZipper();
+        $cache = static::createMock(FilesystemAdapter::class);
+        $cache->expects(static::any())
```

Comment:
bit ugly to test when we type hint on a specific adapter, and tbh not sure how to test a cache hit doesn't compute